src/main/java/it/gov/pagopa/admissibility/config/WebClientConfig.java

```diff
@@ -10,31 +12,58 @@
 import org.springframework.web.reactive.function.client.WebClient;
 import reactor.netty.http.client.HttpClient;
 
+import java.io.IOException;
+import java.io.InputStream;
 import java.time.Duration;
 import java.util.concurrent.TimeUnit;
 
+import static it.gov.pagopa.admissibility.rest.anpr.AnprWebClient.*;
```

Comment:
move in this WebClientConfig class utilities function instead of reference to Anpr classes